src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte

```diff
@@ -32,15 +32,15 @@
 <PageTitle>{$i18n.dapps.text.title}</PageTitle>
 
 {#if nonNullish(featuredDapp) && nonNullish(featuredDapp.screenshots)}
-	<div class="mb-10">
+	<div class="mb-6 md:mb-10">
 		<DappPromoBanner
 			on:click={() => modalStore.openDappDetails(featuredDapp)}
 			dAppDescription={featuredDapp}
 		/>
 	</div>
 {/if}
 
-<div class="flex flex-wrap gap-4">
+<div class="no-scrollbar flex gap-4 overflow-x-auto p-1 md:flex-wrap md:p-0">
```

Comment:
with `overflow-auto` (including y), does the scrollbar appear? we don't want the same behavior as before?

Reply:
We explicitly do not want a scrollbar here on mobile => that's the reason for the no-scrollbar class. That's why for larger screen flex-wrap will wrap items below.

Reply:
just to clarify, is this something purely for this component or in general? for the entire OISY? my question is the same as above, shouldn't we treat this issue at top-level?

Reply:
Shouldn't we leave it up to the implementation to show a scrollbar if needed? At least until there is a clear pattern. As of now we only have this one horizontal scrollbar - I think it's too early to optimise and generalise. That's why I introduced the no-scrollbar class.